scripts/memory_logger.py

```diff
@@ -75,36 +57,36 @@ def _parseStackTrace(self, gibberish):
     else:
       return 'Stack Trace failed:', gibberish
 
-  def _waitForResponse(self, wait=True):
-    while wait:
-      self.gdb_stdout.seek(self.last_position)
-      for line in self.gdb_stdout:
+  def _waitForResponse(self, gdb_stdout, delta=float(5)):
```

Comment:
I was going to allow for other defines to adjust the maximum wait in seconds, before giving up. But as it turns out, only one define is really interested in waiting for a response from gdb. So I am just going to make this private instead

```python
  def _waitForResponse(self, gdb_stdout):  
    end_queue = time.time() + float(5)
    while time.time() < end_queue:
```